### DISCUSSION TITLE
Ghost 博客的数据导入功能

### DISCUSSION THREAD

**Discussion by rebron1900** [halo-dev/halo#4795] [Ideas]
### Your current Halo version

暂时没使用

### Describe this feature

希望能导入Ghost博客的数据，Ghost博客的数据为JSON格式，我看了下大体数据结构其实和Halo是差不多一致的。
数据格式文件如下。
[ghost.json.txt](https://github.com/halo-dev/halo/files/8448389/ghost.json.txt)

**Comment by Jesse-hui** [comment]
1

**Comment by JohnNiang** [comment]
Hi @rebron1900 ，建议移除掉 JSON 中的隐私信息后重新上传。同时建议修改已经暴露的隐私数据。
